solidity/README.md

```diff
@@ -28,7 +28,7 @@ yarn test
 
 ### Fixtures
 
-Some forge tests may generate fixtures in the [fixtures](./fixtures/) directory. This allows [SDK](../typescript/sdk) tests to leverage forge fuzzing. These are git ignored and should not be committed.
```

Comment:
These relative links don't work when this readme is used for the npm listing page: https://www.npmjs.com/package/@hyperlane-xyz/core